A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/AltimeterIndicator.js

```diff
@@ -1101,26 +1102,40 @@ class Jet_PFD_AltimeterIndicator extends HTMLElement {
             bg.setAttribute("width", _width.toString());
             bg.setAttribute("height", _height.toString());
             bg.setAttribute("fill", "#343B51");
-            bg.setAttribute("stroke-width", "4");
+            bg.setAttribute("stroke-width", "3");
             this.centerSVG.appendChild(bg);
+
             var topLine = document.createElementNS(Avionics.SVG.NS, "line");
             this.topLine = topLine;
             topLine.setAttribute("x1", _left.toString());
             topLine.setAttribute("y1", (_top + 2).toString());
             topLine.setAttribute("x2", (_left + _width + arcWidth).toString());
             topLine.setAttribute("y2", (_top + 2).toString());
             topLine.setAttribute("stroke", "white");
-            topLine.setAttribute("stroke-width", "4");
+            topLine.setAttribute("stroke-width", "3");
             this.centerSVG.appendChild(topLine);
+
             var bottomLine = document.createElementNS(Avionics.SVG.NS, "line");
             this.bottomLine = bottomLine;
             bottomLine.setAttribute("x1", _left.toString());
             bottomLine.setAttribute("y1", (_top + _height - 2).toString());
             bottomLine.setAttribute("x2", (_left + _width + arcWidth).toString());
             bottomLine.setAttribute("y2", (_top + _height - 2).toString());
             bottomLine.setAttribute("stroke", "white");
-            bottomLine.setAttribute("stroke-width", "4");
+            bottomLine.setAttribute("stroke-width", "3");
             this.centerSVG.appendChild(bottomLine);
+
+            // JZ: vertical line on the scale
+            var verticalLine = document.createElementNS(Avionics.SVG.NS, "line");
```

Comment:
Use `const`.

Reply:
Would appreciate if you explain why

Reply:
`var` is a function-scoped variable that exists everywhere in the code, which is bad practice.

Only `let` and `const` should be used now. Both are block-scoped, meaning you can safely use them with them same name in different if statements.

Since you never change the value of the variable, it should be `const`. This indicates to other developers that it is not a value that will change later in the code, and prevents misuse.

`const` should be preffered unless it is necesssary to use `let`. `var` should never be used.

Hope this helps ! 🙂

Reply:
> cause teh other elements of the original code use var

Yeah, the original code breaks a lot of conventions. `var` being one of them.